drivers/espi/espi_npcx.c

```diff
@@ -147,6 +147,20 @@ static const struct npcx_vw_out_config vw_out_tbl[] = {
 	NPCX_DT_VW_OUT_CONF(ESPI_VWIRE_SIGNAL_SUS_ACK, vw_sus_ack),
 };
 
+/*  Virtual wire GPIOs for platform level usage (High at Reset state) */
+static const struct npcx_vw_out_config vw_out_gpio_tbl1[] = {
```

Comment:
Is there going to be more tables? (wondering why ..._tbl1 and not just _tbl)

Reply:
from the eSPI spec 
1. below signals are high at reset state
SLV_GPIO_VW_<0..15>
SLV_GPIO_VW_<32..47>
2. below signals are low at reset state
SLV_GPIO_VW_<16..31>
SLV_GPIO_VW_<48..63>

Only initial 8 signals are being used at present hence tbl1. We can create tabl2 once we start using rest of the signals